config.mk

```diff
@@ -8,9 +8,15 @@ MANPREFIX = ${PREFIX}/share/man
 
 # flags
 CPPFLAGS += -D_DEFAULT_SOURCE -DVERSION=\"${VERSION}\"
-CFLAGS   += -g --std=gnu99 -pedantic -Wall -Wno-overlength-strings -Os ${STATIC} ${CPPFLAGS}
+CFLAGS   += --std=gnu99 -pedantic -Wall -Wno-overlength-strings ${STATIC} ${CPPFLAGS}
 LDFLAGS  += -lm -L${X11LIB}
 
+ifeq (Release, ${BUILD_TYPE})
+CFLAGS += -Os
```

Comment:
Since now we've been including the debug symbols in the release build (`-g`) and noone has complained so far. Any specific reason for removing them? - This is not a required change just looking for some justification.